lib/blpapi-wrapper.ts

```diff
@@ -373,11 +371,9 @@ export class Session extends events.EventEmitter {
                 //      until ES6 Map is available
                 this.subscriptions[cid] = s;
 
-                // Lookup by service the events we need to listen to.
-                if (!(serviceUri in SERVICE_TO_SUBSCRIPTION_EVENTS_MAP)) {
-                    throw new Error('Unknown service: ' + serviceUri);
-                }
-                var events: string[] = SERVICE_TO_SUBSCRIPTION_EVENTS_MAP[serviceUri];
+                assert(serviceUri in SERVICE_TO_SUBSCRIPTION_EVENTS_MAP,
+                       util.format('Service, %s, not handled', serviceUri));
```

Comment:
Either way is fine by me, if the stack gets printed it'll be clear enough too. I generally like the message being super clear (C habit, I guess, where the stack trace isn't always that easy to read) but it'll be visible either way in this case.